rasa/core/training/story_reader/markdown_story_reader.py

```diff
@@ -29,11 +29,14 @@ class MarkdownStoryReader(StoryReader):
 
     """
 
-    async def read_from_file(self, filename: Text,) -> List[StoryStep]:
+    async def read_from_file(self, filename: Text) -> List[StoryStep]:
         """Given a md file reads the contained stories."""
 
         try:
-            with open(filename, "r", encoding=io_utils.DEFAULT_ENCODING) as f:
+            # TODO: Hack by Genie for Windows
```

Comment:
why does it work currently for Windows?

Reply:
@evgeniiaraz The MD stuff can be undone as it was just a temporary hack and we just need yaml

Reply:
let's keep for now

Reply:
@Ghostvv even if keep markdown, the e2e stuff shouldn't make into master as the format is completely off charts

Reply:
what do you mean by `format is completely off charts`?